### ISSUE TITLE
XML Parser Fails

### ISSUE THREAD

**Issue by ZeeshanSultan** [xdevl/node-apk#7]
XML Parser fails to parse the application section for some apps i.e

- https://m.apkpure.com/capcut-video-editor/com.lemon.lvoverseas
- https://m.apkpure.com/gta-5/com.ImpulseGaming.GTAVMobile

Fails on fetching app labels because XML may be parsed incorrectly
```
 const resources_info = await apk.getResources();
 const label = resources_info.resolve(manifest_info.applicationLabel);
```

```
/Users/test/Desktop/apktest/node-apk/build/lib/android.js:106
            return this.xml.children.application[0].attributes.label;
                                                ^

TypeError: Cannot read properties of undefined (reading '0')
    at Manifest.get (/Users/test/Desktop/apktest/node-apk/build/lib/android.js:106:49)
    at parsed (/Users/test/Desktop/apktest/app.js:17:56)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by xdevl** [comment]
Hi there, thank you for reporting the issue. Parsing seems to work fine for the second APK (https://m.apkpure.com/gta-5/com.ImpulseGaming.GTAVMobile). I can however reproduce the problem with the first one you shared, I will look into it.

**Comment by xdevl** [comment]
Fix for this went into 1.2.1 and has now been published on npmjs.com
